docs/subnets/deploy-a-gnosis-safe-on-your-evm.md

```diff
@@ -0,0 +1,827 @@
+# Deploy a Gnosis Safe on Your Subnet EVM
+
+## Introduction
+This article shows how to deploy and interact with a [Gnosis Safe](https://gnosis-safe.io/) programmatically on any [Subnet EVM](README.md).
+
+If you are looking for more information regarding the Gnosis Safe protocol, please check out [these developer docs](https://docs.gnosis-safe.io/).
+
+## Prerequisites
+This tutorial assumes that:
+
+- A Subnet and EVM blockchain has been created. Avalanche tools allow users to do this on [Mainnet](../subnets/subnet-cli), [Fuji](../subnets/create-a-fuji-subnet) or a [Local network](../subnets/create-a-local-subnet).
+- Your node is currently validating your target Subnet.
+- Your wallet has a balance of the Subnet native token(Specified under _alloc_ in your [Genesis File](./customize-a-subnet.md#genesis)).
+
+
+The entirety of this tutorial will require you to work with 3 projects (4 if running locally)
+- [safe-contracts](https://github.com/safe-global/safe-contracts.git)
+- [safe-tasks](https://github.com/5afe/safe-tasks.git)
+- [avalanche-smart-contract-quickstart](https://github.com/ava-labs/avalanche-smart-contract-quickstart)
+- [avalanche-network-runner](../quickstart/create-a-local-test-network#avalanche-network-runner) (Local Workflow)
+
+## Custom Network Workflow
+
+### Setup
+Set up the safe-contracts repository by running the following Commands:
+
+```zsh
+git https://github.com/safe-global/safe-contracts.git
+cd safe-contracts
+yarn
+```
+
+Next, change `.env.example` to `.env` and set the variable,`PK` to your wallet's _private key_. Here, we can also add our node's RPC endpoint as our `NODE_URL`. 
+
+Example:
+```env
+PK="<YOUR-PRIVATE-KEY-HERE>"
+PK2=""
+INFURA_KEY=""
+# Used for custom network
+NODE_URL="<YOUR-SUBNET-NODE-RPC-URL-HERE>"
+```
+
+Next, add your Subnet Network parameters to [`hardhat.config.ts`](https://github.com/safe-global/safe-contracts/blob/main/hardhat.config.ts):
+
+```ts
+networks: {
+  subnet: {
+    url: `${NODE_URL}`,
+    chainId: 99999,
+    gasPrice: "auto",
+    accounts: [`${PK}`, ],
+  },
+}
+```
+
```

Comment:
You can use :::note as in https://docusaurus.io/docs/markdown-features/admonitions